qemu/tests/vioser_in_use.py

```diff
@@ -45,6 +46,22 @@ def live_migration_guest(test, params, vm, session):
     vm.migrate()
 
 
+@error_context.context_aware
+def vcpu_hotplug_guest(test, params, vm, session):
+    """
+    Vcpu hot plug test.
+    """
+
+    maxcpus = int(params["vcpu_maxcpus"])
+    current_cpus = int(params.get("smp", 2))
+    cpu_hotplug_cmd = params.get("cpu_hotplug_cmd", "")
```

Comment:
I think this is unnecessary in your case, leave `plug_command` as its default value when you call `vm.hotplug_vcpu()` would be okay.

Reply:
yes, will update.